### ISSUE TITLE
NoisyAnd issue

### ISSUE THREAD

**Issue by ChenYuhang243** [GA-1101/m6A-Maize#1]
I currently use NoisyAnd layer in my model, but the training loss doesn't decrease. I wonder if u encountered this kind of problem?
And the original paper used a joint crossentropy, code in the custom layer does't show that.
